ruby/ruby_releases/ruby_benchmarks/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Anyway after this changes, I'll merge and fix anything I see amiss. Please start working to get benchmark comparison working on RubyBench web